core-services/prow/02_config/openshift/prometheus-community-windows_exporter/_prowconfig.yaml

```diff
@@ -0,0 +1,12 @@
+tide:
```

Comment:
Don't you also need a `_pluginconfig.yaml` with:
```yaml
approve:
- commandHelpLink: https://go.k8s.io/bot-commands
  repos:
  - openshift/prometheus-community-windows_exporter
  require_self_approval: false
plugins:
  openshift/prometheus-community-windows_exporter:
    plugins:
    - approve
```
? Or is that not needed?

Reply:
@aravindhp _pluginconfig.yaml is present in this commit-> https://github.com/openshift/release/blob/5c69c25159d11e9b9078c7a3697171ff675afbea/core-services/prow/02_config/openshift/prometheus-community-windows_exporter/_pluginconfig.yaml

Reply:
Oops, sorry I missed that.